cli/src/config.rs

```diff
@@ -122,8 +123,23 @@ pub fn read_configuration(path: &Path) -> Result<Config> {
     };
 
     // If an api token has been set in the environment, prefer that
-    if let Ok(key) = env::var("PHYLUM_API_KEY") {
-        config.auth_info.offline_access = Some(RefreshToken::new(key));
+    match env::var("PHYLUM_API_KEY") {
+        Ok(key) if !key.is_empty() => {
+            config.auth_info.offline_access = Some(RefreshToken::new(key));
+        },
+        Ok(_) => {
+            print_user_warning!("Ignoring empty PHYLUM_API_KEY");
+        },
```

Comment:
It might be personal preference (and feel free to ignore this comment). But since this is a match statement, you can use `Ok("")` explicitly rather than using a guard.

```suggestion
        Ok("") => {
            print_user_warning!("Ignoring empty PHYLUM_API_KEY");
        },
        Ok(key) => {
            config.auth_info.offline_access = Some(RefreshToken::new(key));
        },
```